src/main.rs

```diff
@@ -47,7 +47,7 @@ struct Opts {
 #[derive(Debug, Subcommand)]
 enum Commands {
     /// Initialize Broker configuration.
-    Init,
+    Init(config::RawBaseArgs),
```

Comment:
**[required]**
Please use a separate args struct for init (`RawInitArgs` / `InitArgs` ?), one that only accepts the args we intend init to accept.

**[optional]**
From there:
- `RawBaseArgs` should probably become `RawRunArgs`
- `BaseArgs` should probably become `RunArgs`
- `config::validate_args` should probably become `RawRunArgs::validate`.
- This then allows `config::validate_init_args` to become `RawInitArgs::validate`.